db/migrate/20221030123900_add_version_to_exercise_representations.rb

```diff
@@ -0,0 +1,14 @@
+class AddVersionToExerciseRepresentations < ActiveRecord::Migration[7.0]
+  def change
+    add_column :exercise_representations, :representer_version, :smallint, null: false, default: 1
```

Comment:
I like `:smallint` a lot better than `:integer, limit: 2`. We should do that more!